### ISSUE TITLE
testing: no artifacts in `TestSqlLiteLogic`

### ISSUE THREAD

**Issue by asubiotto** [cockroachdb/cockroach#51207]
When TestSqlLiteLogic fails, the only information is in the build log which is hard to parse. The artifacts tab is empty (e.g. https://github.com/cockroachdb/cockroach/issues/51134).

**Comment by yuzefovich** [comment]
Some discussion in [this thread](https://cockroachlabs.slack.com/archives/CJ0H8Q97C/p1608588075435800).

**Comment by yuzefovich** [comment]
Tobi found an issue [here](https://cockroachlabs.slack.com/archives/CJ0H8Q97C/p1611158110069300?thread_ts=1608588075.435800&cid=CJ0H8Q97C), so I updated the TC config.

**Comment by yuzefovich** [comment]
Seems stale and unactionable.
